src/java.base/share/classes/java/lang/String.java

```diff
@@ -1309,29 +1313,29 @@ private static int malformed4(byte[] src, int sp) {
     }
 
     @SuppressWarnings("unchecked")
-    private static <E extends Exception> E malformedInputException(int off, int nb, Class<E> exceptionType) {
+    private static <E extends Exception> E malformedInputException(int off, int nb, Class<E> exceptionClass) {
```

Comment:
The argument name can be "_", since it is ignored and is unused in the body.
Comment the method as always throwing MalformedInputException.
Similarly in the other malformedInput methods*.

Reply:
I've removed the need to pass a `Class<E>` to these exception factories, and improved their documentation in 584e6e213ec.

> The argument name can be "_", since it is ignored and is unused in the body.

For the record, AFAIK, this is not possible for method arguments. Quoting [§6.1 (Names > Declarations)](https://docs.oracle.com/javase/specs/jls/se24/html/jls-6.html#jls-6.1):

> If a declaration does not include an identifier, but instead includes the keyword _ (underscore), then the entity cannot be referred to by name. The following kinds of entity may be declared using an underscore:
>
> * A local variable, one of the following: ...
> * An exception parameter of an exception handler declared in a catch clause of a try statement
> * A formal parameter of a lambda expression